.github/workflows/publish.yml

```diff
@@ -0,0 +1,29 @@
+name: Copy folder to other branch
+
+on:
+  push:
+    branches:
+      - issue#5-gen-web
+
+jobs:
+  copy:
+    name: Copy my folder
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: copy
+        env:
+          SRC_FOLDER_PATH: 'optimized'
+          TARGET_BRANCH: 'gh-pages'
+        run: |
+          files=$(find $SRC_FOLDER_PATH -type f) # get the file list
```

Comment:
This should be moved to a shell script in the repo, and run as a single command from there.